exporters/jaeger/src/test/java/io/opentelemetry/exporters/jaeger/JaegerIntegrationTest.java

```diff
@@ -53,13 +55,22 @@
 
   @SuppressWarnings("rawtypes")
   @ClassRule
-  public static GenericContainer jaeger =
-      new GenericContainer("jaegertracing/all-in-one:" + JAEGER_VERSION)
-          .withExposedPorts(COLLECTOR_PORT, QUERY_PORT)
-          .waitingFor(new HttpWaitStrategy().forPath("/"));
+  @Nullable
+  public static GenericContainer jaeger = null;
+
+  static {
+    // make sure that the user has enabled the docker-based tests
+    if (Boolean.parseBoolean(System.getProperty("enable.docker.tests"))) {
+      jaeger =
+          new GenericContainer<>("jaegertracing/all-in-one:" + JAEGER_VERSION)
+              .withExposedPorts(COLLECTOR_PORT, QUERY_PORT)
+              .waitingFor(new HttpWaitStrategy().forPath("/"));
+    }
+  }
 
   @Before
   public void setupJaegerExporter() {
+    Assume.assumeNotNull(jaeger);
```

Comment:
It is confusing to have this in Before, and probably before is not needed since only one test. I would remove `Before` from this method and explicitly call it in the only test we have. Then the `Assume` is only in the test method.

Reply:
implemented. please take a look